facade/initial/initialNodeFacade_test.go

```diff
@@ -6,9 +6,30 @@ import (
 
 	"github.com/ElrondNetwork/elrond-go-core/core/check"
 	"github.com/ElrondNetwork/elrond-go-core/data/api"
+	"github.com/ElrondNetwork/elrond-go/facade"
+	"github.com/ElrondNetwork/elrond-go/testscommon"
 	"github.com/stretchr/testify/assert"
 )
 
+func TestDisabledNodeFacade(t *testing.T) {
```

Comment:
might rename all the tests TestDisabled* => TestInitial*

Reply:
done